python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -181,14 +210,87 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
             {
               label: "Required resources",
               value:
-                Object.entries(actor.requiredResources).length > 0 &&
-                Object.entries(actor.requiredResources)
-                  .sort((a, b) => a[0].localeCompare(b[0]))
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", "),
+                Object.entries(actor.requiredResources).length > 0
+                  ? Object.entries(actor.requiredResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : "",
             },
           ];
 
+    // Move some fields to the back and de-prioritize them.
+    const pushFieldsToBack = [
+      "NumObjectIdsInScope",
+      "NumLocalObjects",
+      "UsedLocalObjectMemory",
+    ];
+    pushFieldsToBack.forEach((fieldName) => {
+      const foundIdx = rawInformation.findIndex(
+        (info) => info.label === fieldName,
+      );
+      if (foundIdx !== -1) {
+        const foundValue = rawInformation[foundIdx];
+        rawInformation.splice(foundIdx, 1);
+        rawInformation.push(foundValue);
+      }
+    });
+
+    // Apply transformation to add styling for information field
+    const transforms: Record<string, (info: ActorInformation) => ReactNode> = {
+      Pending: (info: ActorInformation) => {
+        if (actor.state !== -1 && actor.taskQueueLength >= 50) {
+          return (
+            <span
+              className={classes.warningTooManyPendingTask}
+              key={info.label}
+            >
+              {info.label}: {info.value}
+            </span>
+          );
+        } else {
+          return info.label;
+        }
+      },
+      ActorTitle: (info: ActorInformation) => (
+        <span className={classes.actorTitle} key={info.label}>
+          {info.value}
+        </span>
+      ),
+      NumObjectIdsInScope: (info: ActorInformation) => (
+        <span className={classes.secondaryFieldsHeader} key={info.label}>
+          <br></br>
```

Comment:
I realize this is to separate the second group of attributes from the main group, but is there any way we can avoid using a `<br>` here to make this more robust to any reordering of `pushFieldsToBack` in the future? Maybe we could transform all the info into React nodes, then split into two groups below and optionally render the second group in a separate (block) element if it's non-empty? What do you think?